src/components/CardsList/CardsList.tsx

```diff
@@ -0,0 +1,14 @@
+import Card from '../Card/Card';
+import { products } from '../../data/products';
+
+import Wrapper from './CardsList.styles';
+
+export default function CardsList() {
```

Comment:
Select one style to follow everywhere. Compare with the style you have for Card:

```ts
function Card({ product }: CardProps) {
  return <CardWrapper>{product.name}</CardWrapper>;
}

export default Card;
```

Reply:
fixed